packages/daemon/src/mail.js

```diff
@@ -41,16 +42,20 @@ export const makeMailboxMaker = ({
     };
 
     /**
-     * @param {string} petName
+     * @param {string[]} petNamePath - A sequence of pet names.
```

Comment:
Does the tsdoc need an ellipses somewhere?

Reply:
TIL you do the ellipses in the JSDoc type annotations. (I also forgot to add them to `types.d.ts`)